client/package.json

```diff
@@ -74,6 +74,10 @@
     "@genesislcap/eslint-config": "{{versions.UI}}",
     "@genesislcap/foundation-testing": "{{versions.UI}}",
     "@genesislcap/genx": "{{versions.UI}}",
+    "@genesislcap/eslint-stylelint-builder": "{{versions.UI}}",
```

Comment:
ahh now I see! Makes sense related to the other Genx PR, so "root" package.json for client/apps should list the desired builders

Reply:
Yeah, what's more, if for example, they're using their own bundler, they don't need to list the Genesis ones. Previously those were installed regardless.